.github/workflows/prod-promote.yml

```diff
@@ -0,0 +1,19 @@
+### This is the Terraform-generated prod-promote.yml workflow for the alma-webhook-lambdas-prod app repository ###
+name: Prod Promote Lambda Container
+on:
+  workflow_dispatch:
```

Comment:
Just confirming, this will run automatically when we tag a release but we can also promote by manually running the action?

Reply:
Correct! I have included the `workflow_dispatch` option for all three workflows to allow devs to run them manually through the GitHub UI if needed.